src/site/content/en/fast/serve-responsive-images/index.md

```diff
@@ -21,6 +21,24 @@ Serving desktop-sized images to mobile devices can use 2–4x more data than
 needed. Instead of a "one-size-fits-all" approach to images, serve different
 image sizes to different devices.
 
+## Responsive images and Core Web Vitals
+
+When you serve responsive images, you're evaluating the display capabilities of
+the user's device and choosing one of a set of image candidates that are optimal
+for display based on those criteria. The result&mdash;as stated previously&mdash;is
+rather than delivering too much image data to devices that won't benefit from it,
+you're serving an appropriately sized image for the device. For smaller devices such
+as phones and tablets, this equates to reduced data usage as compared to devices
+with larger screens, such as laptops.
+
+The effects of this on performance go beyond faster loading times for images. For
```

Comment:
Not sure about this first sentence. "It's more than just faster loading images. For example you can have a faster loading LCP image". That's just a faster loading image isn't it?

I do agree with the phrase, but think we should give a better example (while also leaving the important LCP example in place).

What about saying it can affect the load time of the rest of the page as large images can hold up all other resources due to bandwidth constraints? And it also impacts Core Web Vitals. For example if your LCP...

Reply:
The point I guess I was (poorly) trying to make is that the effect extends to the LCP metric, which is a measure of perceptual performance. I'll have to workshop this.

Reply:
I revised the language in this paragraph, @tunetheweb.